app/forms/coc/DivorceSelectYearForm.scala

```diff
@@ -66,7 +66,7 @@ class DivorceSelectYearForm@Inject()(applicationConfig: ApplicationConfig, timeS
     case _ => Invalid(ValidationError("pages.divorce.date.error.mandatory"))
 
   }
-
+  //TODO - Refactor to address warning
```

Comment:
Has this been done

Reply:
This was added following a discussion with Kurt, will likely result in a separate ticket, I'll raise further discussion with him.